packages/dcos-integration-test/extra/resiliency/test_aws_cf.py

```diff
@@ -0,0 +1,75 @@
+import logging
```

Comment:
The only way I could think of doing that without mucking with pytest internal hooks is to drop this all in a test class. I could do that, but Im not a huge fan of test classes